A51-custom-backend-metrics.md

```diff
@@ -160,23 +163,27 @@ Out-of-band data will be sent only if the client starts a streaming call for the
 If the backend is designed to provide this information but the client never asks for it, 
 it will not be used. Conversely, if the backend does not provide the information but the client 
 does start a call asking for it, the backend will terminate the call with status `UNIMPLEMENTED`, 
-in which case the client will not retry the call.
+in which case the client must not retry the call on the same connection.
 
-Any LB policy that is interested in this out-of-band data may subscribe to it. The client will 
+Any LB policy that is interested in this out-of-band data may subscribe to it.
+there will be an API that allows LB policies to subscribe to the out-of-band data. The client will
 maintain an open `OpenRCAService/StreamCoreMetrics` call whenever there is at least one LB policy 
 subscribing to the data. When a policy subscribes, it will indicate the interval at which it would like to get updates. 
 If multiple policies subscribe with different intervals, we will use the minimum of those intervals. 
 Note that the ORCA protocol involves the client telling the server the interval at which 
 it wants reports when the call starts, so if the client needs to change its requested interval, 
 it will need to cancel and restart the call.
-A client may change its requested interval at any time; this is expected to be a fairly rare operation 
+The API provided to LB policies needs to allow changing its requested interval at any time;
+this is expected to be a fairly rare operation
 (usually triggered by a config change for one of the subscribing LB policies),
 It is important that a load balancing policy is able to re-config the frequency
 and should not require heavy-weight operations like reconnecting to backends or recreating subchannels or LB policies.
 
 Each policy that has subscribed to the data will get a callback whenever a new message is sent to the client on the call.
 As with per-request data, the protobuf deserialization will happen only once.
 The data provided to the callback should be exposed to the LB policy using the same type as used for per-request data.
+Currently, out-of-band reporting only supports utility metrics, details see the Rationale section,
```

Comment:
"details see the Rationale section" grammatically broken/weird in a couple ways. Would it be better as a parenthetical `(see Rationale section)`